config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml

```diff
@@ -0,0 +1,32 @@
+periodics:
+- interval: 3h
```

Comment:
Maybe this interval should be longer. Conformance tests take a while.

Reply:
3h seems fine, that's also the cadence for the existing job.